.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+### Description
+What does this PR do?
+
+### TODO if you're a community member
+- [ ] Make sure there's a changelog. This is a description of what changed from
```

Comment:
Do we often have to ask community members to fix their changelog formatting? Perhaps linking to [this part](https://github.com/hashicorp/vault/blob/main/CONTRIBUTING.md#changelog-entries) of CONTRIBUTING.md would be helpful here since it shows exactly what a changelog should look like.

Reply:
It's not uncommon. A lot of community PRs do include a changelog, but sometimes on old PRs I sometimes just add the changelog for them (since asking them to come back after months or years to add it can feel a little wrong).

Still, I think that linking to the contributing.md would be valuable, since this description doesn't say what a changelog is. I do wonder if this kind of information should go in the failing changelog GHA instead, though. I think it's best (where possible) to not duplicate

Here's an example output from a failing changelog entry today:
```
looking for changelog file changelog/25080.txt
Not found.
looking for changelog file matching changelog/_go-ver-*.txt
Not found.

Did not find a changelog entry named changelog/25080.txt
If your changelog file is correct, skip this check with the 'pr/no-changelog' label
Reference - https://github.com/hashicorp/vault/pull/10363 and https://github.com/hashicorp/vault/pull/11894
```

Reply:
In addition, we often have to remind community members to add the changelog file. I think we should either link to the contributing.md for changelog specifics, because many folks don't know how we want that done, how it should be formatted, and the like. The wording here might also be too vague in general, and I'll noodle a bit on how we might make it clearer for the community.

Reply:
I've updated this item to include a link to those changelog instructions.